podpac/datalib/terraintiles.py

```diff
@@ -0,0 +1,522 @@
+"""
+Terrain Tiles
+
+Hosted on AWS S3
+https://registry.opendata.aws/terrain-tiles/
+
+Description
+    Gridded elevation tiles
+Resource type
+    S3 Bucket
+Amazon Resource Name (ARN)
+    arn:aws:s3:::elevation-tiles-prod
+AWS Region
+    us-east-1
+
+Documentation: https://mapzen.com/documentation/terrain-tiles/
+
+Attribution
+-----------
+- Some source adapted from https://github.com/tilezen/joerd
+
+Attributes
+----------
+BUCKET : str
+    AWS S3 bucket
+TILE_FORMATS : list
+    list of support tile formats
+
+Notes
+-----
+
+See https://github.com/racemap/elevation-service/blob/master/tileset.js
+for example skadi implementation
+"""
+
+import os
+import re
+from itertools import product
+import logging
+
+import traitlets as tl
+import boto3
+from botocore.handlers import disable_signing
+import numpy as np
+import rasterio
```

Comment:
I think again rasterio is optional... I'll check and fix.

Reply:
Yeah, and boto3/botocore, too.